test/integration/single_tunnels/inspec.yml

```
@@ -2,4 +2,4 @@ name: single_tunnels
 depends:
   - name: inspec-gcp
     git: https://github.com/inspec/inspec-gcp.git
-    version: '~> 0.9.0'
+    version: '~> 0.11.0'
```

`version` is not actually supported by the git dependency.

```suggestion
    tag: v0.11.0
```